<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>7</y>
      <width>280</width>
      <height>365</height>
     </rect>
    </property>
    <property name="title">
     <string>Доходы/Расходы</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_Income_2">
             <property name="text">
              <string>Общий доход</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Общий расход</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Остаток</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Остаток с планами</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_Income">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Spending">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Total">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_TotalWithFuture">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>руб</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>руб</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>руб</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>руб</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="clear_ListWirdget">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>280</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Планы</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QListWidget" name="listWidget_Plans"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Будущие затраты</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FutureSpending">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>руб</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="add_To_MainList">
         <property name="text">
          <string>Перенести </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transferAll">
         <property name="text">
          <string>Перенести всё</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeletePlan">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_list_Plans">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>210</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Категория</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_Category"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Цена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_Price"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_Add">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Cansel">
         <property name="text">
          <string>Отмета</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addToPlans">
         <property name="text">
          <string>Отложить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Сохранить</string>
     </property>
     <addaction name="Save_Current"/>
     <addaction name="Save_Previous"/>
     <addaction name="SaveFuturePlans"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Загрузить</string>
     </property>
     <addaction name="DownloadCurrent"/>
     <addaction name="DownoadPrevious"/>
     <addaction name="DownloadFuturePlans"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="Instruction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="Save_Current">
   <property name="text">
    <string>В текущий месяц</string>
   </property>
  </action>
  <action name="Save_Previous">
   <property name="text">
    <string>В прошлый месяц</string>
   </property>
  </action>
  <action name="SaveFuturePlans">
   <property name="text">
    <string>В планы</string>
   </property>
  </action>
  <action name="DownloadCurrent">
   <property name="text">
    <string>Текущий месяц</string>
   </property>
  </action>
  <action name="DownoadPrevious">
   <property name="text">
    <string>Прошлый месяц</string>
   </property>
  </action>
  <action name="DownloadFuturePlans">
   <property name="text">
    <string>Планы</string>
   </property>
  </action>
  <action name="Correct_Input">
   <property name="text">
    <string>Как правильно вводить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Руководство</string>
   </property>
  </action>
  <action name="AboutProgramm">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="Instruction">
   <property name="text">
    <string>Руководство</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
